controller/filehandlingcontroller.php

```diff
@@ -143,7 +146,15 @@ public function save($path, $filecontents, $mtime) {
 				// File same as when opened, save file
 				if($this->view->isUpdatable($path)) {
 					$filecontents = iconv(mb_detect_encoding($filecontents), "UTF-8", $filecontents);
-					$this->view->file_put_contents($path, $filecontents);
+					try {
+						$this->view->file_put_contents($path, $filecontents);
+					} catch (LockedException $e) {
+						return new DataResponse(['message' => $e->getMessage()], Http::STATUS_BAD_REQUEST);
```

Comment:
Sure that this is no regression of https://github.com/owncloud/files_texteditor/pull/96?